providers/sadad/models.go

```diff
@@ -0,0 +1,118 @@
+package sadad
+
+import (
+	"github.com/GoFarsi/paygap/client"
+	"time"
+)
+
+type MultiplexingType int64
```

Comment:
change `MultiplexingType` type to int for enum.

```go
type MultiplexingType int

```

Reply:
Done.